### PR TITLE
Added a test simpleSearchTest

### PR DESCRIPTION
**Pull request by DmitriPanichin**
Added a test simpleSearchTest

### CODE REVIEW

File: src/test/java/school/redrover/LocomotiveGroupTest.java [hunk 1]
```diff
@@ -1,58 +1,65 @@
...
 import org.testng.annotations.Test;
 
-import school.redrover.runner.BaseTest;
-
-
+import java.time.Duration;
 import java.util.concurrent.TimeUnit;
 
-
-public class LocomotiveGroupTest extends BaseTest {
```
Comment (tereshenkov29) [inline src/test/java/school/redrover/LocomotiveGroupTest.java:16]: why "extends BaseTest" is removed?

File: src/test/java/school/redrover/LocomotiveGroupTest.java [hunk 2]
```diff
@@ -1,58 +1,65 @@
...
         String fullName = "Tom Jonson";
         String email = "mail@mail.com";
 
-        getDriver().get("https://demoqa.com/text-box");
+        WebDriver driver = new ChromeDriver();
+        driver.get("https://demoqa.com/text-box");
 
-        String pageTitleText = getDriver().findElement(By.className("main-header")).getText();
+        String pageTitleText = driver.findElement(By.className("main-header")).getText();
         Assert.assertEquals(pageTitleText, "Text Box");
```
Comment (tereshenkov29) [inline src/test/java/school/redrover/LocomotiveGroupTest.java:28]: why getDriver() is replaced to driver?

File: src/test/java/school/redrover/LocomotiveGroupTest.java (hunk 1 above)
Comment (DmitriPanichin) [reply src/test/java/school/redrover/LocomotiveGroupTest.java:16]: Looks like, I resolved conflicts, and applied my changes.

Comment (tereshenkov29) [reply src/test/java/school/redrover/LocomotiveGroupTest.java:16]: but how it is related to the "extends BaseTest" part? if I got it right, it should be in every class

**Comment by DmitriPanichin** [comment]
See the comments

**Pull request closed without merging**